controller/src/config/tasks.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from celery import Celery
+from celery.schedules import crontab
+
+
+# setup celery
+celery = Celery('tasks', broker=os.environ.get("RABBITMQ_URL"))
+
+# configure celery
+celery.conf.update(
+    task_serializer='json',
+    accept_content=['json'],
+    result_serializer='json',
+    timezone='America/Chicago',
```

Comment:
here the timezone is 'America/Chicago', but in settings.py its set to UTC. They should probably be the same